ldapadmin/src/test/java/org/georchestra/ldapadmin/ws/ldap/UserLDAPCrudTest.java

```diff
@@ -59,9 +59,9 @@ public void setUp() throws Exception {
 			ltmpl.bind(ctx);
 		} catch (Exception e) {};
 
-		// recreating ou=groups
+		// recreating ou=roles
 		try {
-			DirContextOperations ctx = new DirContextAdapter("ou=groups");
+			DirContextOperations ctx = new DirContextAdapter("ou=roles");
 			ctx.addAttributeValue("objectClass", "top");
 			ctx.addAttributeValue("objectClass", "organizationalUnit");
 			ctx.addAttributeValue("ou", "users");
```

Comment:
I think you should also change `ctx.addAttributeValue("ou", "users");` to `ctx.addAttributeValue("ou", "roles");`